packages/vike-react-apollo/README.md

```diff
@@ -0,0 +1,130 @@
+<!-- WARNING: keep links absolute in this file so they work on NPM too -->
+
+[<img src="https://vike.dev/vike-readme.svg" align="right" height="90">](https://vike.dev)
+[![npm version](https://img.shields.io/npm/v/vike-react-apollo)](https://www.npmjs.com/package/vike-react-apollo)
+
+# `vike-react-apollo`
```

Comment:
I did some polishing for the `vike-react-query` docs: https://github.com/vikejs/vike-react/tree/main/packages/vike-react-query#readme. Maybe we can also apply some of that polish here as well.